code/src/components/app-input/AppInput.tsx

```diff
@@ -0,0 +1,14 @@
+import { TextField, TextFieldProps } from "@mui/material";
+import AppBox from "@/components/app-box/AppBox";
+
+type AppInputProps = TextFieldProps;
+
+const AppInput = ({ value, ...props }: AppInputProps) => {
```

Comment:
Use just `props` without destructuring because value is not used anywhere else apart from props for `TextField`

Reply:
Fixed